src/distilabel/steps/argilla/base.py

```diff
@@ -20,23 +20,24 @@
 from pydantic import Field, PrivateAttr, SecretStr
 
 try:
-    import argilla as rg
+    import argilla_sdk as rg
```

Comment:
@frascuchon What do you think the timeline on renaming `argilla_sdk` >> `argilla` is? Is it worth waiting for the changes in `argilla` so that it's in line?